### DISCUSSION TITLE
Compile on save not working in latest version

### DISCUSSION THREAD

**Discussion by ademc** [serenity-is/Serenity#7046] [Help]
I created a new project with the latest SERENE template. I have realised that compile on save not working on typescript files.
I tried several things but no solution. I tried on different two computers and different visual studio 2022 versions.
I have little old project and working perfect. This ist the working one
```
{
  "name": "serene.web",
  "dependencies": {
    "@serenity-is/corelib": "6.6.6",
    "@serenity-is/sleekgrid": "1.5.5",
    "@types/jquery": "2.0.48",
    "@types/jquery.validation": "1.16.7",
    "@types/jqueryui": "1.12.6",
    "@types/toastr": "2.1.39"
  },
  "devDependencies": {
    "@serenity-is/tsbuild": "6.6.0",
    "@typescript-eslint/eslint-plugin": "^5.59.9",
    "@typescript-eslint/parser": "^5.59.9",
    "eslint": "^8.41.0",
    "eslint-plugin-vue": "^9.14.1"
  },
  "scripts": {
    "build": "npm run tsbuild",
    "prepare": "dotnet build -target:RestoreTypings",
    "tsbuild": "node ./tsbuild.js",
    "tsbuild:watch": "node ./tsbuild.js --watch"
  },
  "private": true,
  "type": "module"
}

```
And this is new and not working one:
```
{
  "name": "serene.web",
  "dependencies": {
    "@serenity-is/corelib": "6.9.6",
    "@serenity-is/sleekgrid": "1.6.3",
    "@types/jquery": "2.0.48",
    "@types/jquery.validation": "1.16.7",
    "@types/jqueryui": "1.12.6",
    "jsx-dom": "8.0.7",
    "preact": "10.15.0"
  },
  "devDependencies": {
    "@serenity-is/tsbuild": "6.9.10",
    "eslint": "^8.56.0"
  },
  "scripts": {
    "build": "npm run tsbuild",
    "prepare": "dotnet build -target:RestoreTypings",
    "tsbuild": "node ./tsbuild.js",
    "tsbuild:watch": "node ./tsbuild.js --watch"
  },
  "private": true,
  "type": "module"
}

```
What is the real problem and how to fix that? Am i have to wait new version or any quick fix enabled?

**Comment by VictorTomaili** [comment]
Check `appsetting.Development.json` for line `"StartNodeScripts": "tsbuild:watch"`
otherwise run command `npm run tsbuild:watch` to watch changes for typescript files. 
The typescript files are already compiled on build automatically.

**Reply by ademc** [reply]
Yes already tried. No solution. Can i find older versions of vsix files?

**Comment by ademc** [comment]
This is the solution.
![image](https://github.com/serenity-is/Serenity/assets/3390438/44de3dd4-00d9-4814-ad45-72e5c97af9e2)

The real roblem is @serenity-is/tsbuild@6.9.10
